### PR TITLE
Theming

### PR DESCRIPTION
**Pull request by sschoe85**
Had a look at different options how to use variables for colors. Decided on an  JS- object rather than using CSS variables in Global styles or using the Theming Options from Emotion, because then I don't have any extra  div wrapping my objects.
Please comment on that. Thank you!

### CODE REVIEW

File: src/utils/theme.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const colors = {
+    gradient: 'linear-gradient(101deg, #78e4ff, #ff48fa)',
+    gradientsecondary: "linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%)"
```
Comment (lmachens) [inline src/utils/theme.js]:
```suggestion
    gradientSecondary: "linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%)"
```

**Review by lmachens — COMMENTED** [review]
> because then I don't have any extra div wrapping my objects.

You don't need any extra div if you use GlobalStyles or the ThemeProvider.

There are some disavantages, if you use a JS object for your styles.
- You can not change the theme later. (if you plan a dark/light mode)
- CSS is not injected in to the styled components. Instead, you create the styles when you execute the JS code the first time. This is a small performance disadvantage.
- No CSS highlighting / typo checks
- More imports

Some advantages for GlobalStyles and maybe ThemeProvider in emotion:
- Injected during runtime
- Automatic vendor prefixes
- Code Highlighting

File: src/utils/theme.js (hunk 1 above)
Comment (sschoe85) [reply src/utils/theme.js]: Thanks for reviewing, I implemented your suggestions :).

File: src/components/GlobalStyles.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ export default function GlobalStyles({ children }) {
             background: #111;
             color: #fff;
             font-family: "Solway", serif;
+            --gradient: linear-gradient(101deg, #78e4ff, #ff48fa);
+            --gradientSecondary: linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%);
```
Comment (lmachens) [inline src/components/GlobalStyles.js]: If there is no good reason that the variables are in `html`, it's a good practice to add the variables to the `:root` pseudo class.
See https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#Basic_usage

File: src/pages/faq.js [hunk 3]
```diff
@@ -31,7 +32,7 @@ function Faq(props) {
         <FaqCard />
         <FaqCard />
       </FaqSection>
-    </GlobalStyles>
+      </div>
```
Comment (lmachens) [inline src/pages/faq.js:42]: Looks like your code formatter is not working correctly here. This could be an indicator that there is an issue in this file.

File: src/pages/faq.js [hunk 4]
```diff
@@ -19,7 +19,8 @@ function Faq(props) {
   `
 
   return (
-    <GlobalStyles>
+    <div>
+    <GlobalStyles/>
```
Comment (lmachens) [inline src/pages/faq.js:29]: You only need `GlobalStyles` once in your app. These styles are global → should apply everywhere.
Thus, a perfect place would be `App.js`.

File: src/pages/faq.js [hunk 5]
```diff
@@ -19,7 +19,8 @@ function Faq(props) {
   `
```
Comment (lmachens) [inline src/pages/faq.js]: Can't comment the correct line, but a few lines above you define a styled component inside your Faq component.
This is not related to this PR.

**Review by lmachens — COMMENTED** [review]
Good job 👍.
A deployment would help to review this pull request because I am not sure if all CSS is required.

File: src/pages/faq.js (hunk 3 above)
Comment (sschoe85) [reply src/pages/faq.js:42]: Will fix this in another branch

File: src/pages/faq.js (hunk 5 above)
Comment (sschoe85) [reply src/pages/faq.js]: I had already fixed this issue in my other branch.

File: src/components/GlobalStyles.js (hunk 2 above)
Comment (sschoe85) [reply src/components/GlobalStyles.js]: Moved the variables to root

File: src/pages/faq.js (hunk 4 above)
Comment (sschoe85) [reply src/pages/faq.js:29]: will move to PageLayout once my header and footer components are ready. For testing purposes I'll leave it here for now.
